tokens/manage/proxy-accounts.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: 代理账户
-description: 学习如何在基于Moonbeam的网络设置代理账户
+title: 设置代理账户
+description: 了解如何在基于Moonbeam的网络上设置代理帐户，以便您可以将钱包安全保存在冷存储中。
 ---
 
```

Comment:
line 24  should use this updated URL /builders/get-started/networks/moonbase/#get-tokens/